Core/src/org/sleuthkit/autopsy/modules/filetypeid/FileTypeDetector.java

```diff
@@ -233,6 +235,23 @@ public String getMIMEType(AbstractFile file) {
                  * Remove the optional parameter from the MIME type.
                  */
                 mimeType = removeOptionalParameter(mimeType);
+                
+                /*
+                 * If Tika recognizes the file signature, then use the file 
+                 * name to refine the type. In short, this is to exclude the 
+                 * mime types that are determined solely by file extension.
+                 * More details in JIRA-4871.
+                 */
+                if (!mimeType.equals(MimeTypes.OCTET_STREAM)) {
+                    ReadContentInputStream secondPassStream = new ReadContentInputStream(file);
+                    try (TikaInputStream secondPassTikaStream = TikaInputStream.get(secondPassStream)) {
+                        tikaType = tika.detect(secondPassTikaStream, file.getName());
+                        mimeType = tikaType.replace("tika-", ""); //NON-NLS
+                        mimeType = removeOptionalParameter(mimeType);
+                    } catch (Exception ex) {
+                        throw ex;
```

Comment:
I don't believe this catch block is necessary

Reply:
Nice catch (haha), I removed it. Thank you!